resources/skins/nudus/html_enclosure_every.html

```diff
@@ -0,0 +1 @@
+ /&nbsp;<a href="%1"><div style="display:none"><!-- cannot be removed -->%2</div><span style="text-transform: uppercase;">%3</span></a>
```

Comment:
What about `/` in the beginning if the file, why is it there?

Reply:
Never mind that. It needs to be there.

Though looks really weird indeed.

I'll do something about it.

Reply:
Fixed.